pkg/config/config/antivirus.go

```diff
@@ -0,0 +1,83 @@
+package config
+
+import "github.com/go-viper/mapstructure/v2"
+
+// AntivirusContextConfig contains the context-specific antivirus settings
+type AntivirusContextConfig struct {
+	Enabled       bool                         `json:"enabled" mapstructure:"enabled"`
+	OnInfected    string                       `json:"on_infected,omitempty" mapstructure:"on_infected"`
+	Notifications AntivirusNotificationsConfig `json:"notifications,omitempty" mapstructure:"notifications"`
+	Actions       map[string][]string          `json:"actions" mapstructure:"actions"`
+}
+
+// AntivirusNotificationsConfig contains notification settings for antivirus
+type AntivirusNotificationsConfig struct {
+	EmailOnInfected bool `json:"email_on_infected" mapstructure:"email_on_infected"`
+}
+
+// GetAntivirusConfig returns the antivirus configuration for a given context.
+// It reads from contexts.<contextName>.antivirus and returns the configuration for the UI.
+func GetAntivirusConfig(contextName string) *AntivirusContextConfig {
+	if config == nil || !config.Antivirus.Enabled {
+		return &AntivirusContextConfig{Enabled: false}
```

Comment:
You might want to assign this to a variable or constant since it's used a few times.